### ISSUE TITLE
Review alt text for WCAG compliance on Projects page

### ISSUE THREAD

**Issue by IsaacDesigns** [hackforla/website#2613]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will tackle the images under the Projects page.

#### Details
Developers identified that some alt texts in the Projects page should be reviewed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items 
- [x] Make a copy of the tab Template [Page Name]- [Status] 
- [x] With the copied tab replace [Page Name] with the page you are reviewing and [Status] with "Audit in Progress". Don't forget to remove brackets "[ ]"
- [x] Add a link of the page your working on at the bottom in the Resources/Instructions section of this issue where it says "Google sheet Tab for your page".
- [x] For each image on the page, identify if the alt text needs to be changed or not using the recourses below and fill out the google sheet. To locate the alt text use the Google Chrome browser, right click "inspect" to view pages code.
- [x] After completing auditing each image change your google sheet tabs [Status] to Audit Complete.
- [x] Review with Design Lead or another member of the team
- [x] Mark spreadsheet tab as QA (e.g., About Page - Audit QA)
- [x] Add note at bottom of issue saying its ready for QA and use the close with comment, so that it goes to the QA column

Product
- [x] Review the work and sign off on it if ready
   - [x] Mark spreadsheet tab as Complete (e.g., About Page - QA Complete)
   - [x] Add Ready for dev leads label
   - [x] Move to New Issue Approval Column

Development
- [x] Create issues for that particular alt text should be created (preferably labelled good first issue) for new developers to fix
- [ ] Close this issue

### Resources/Instructions
[Projects page](https://www.hackforla.org/projects/)
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
[Google sheet instructions](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713)
[Google sheet template](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=129703697)
[Google sheet Tab for your page](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=68583811)

**Comment by ememercy21** [comment]
provide update
1. Progress
2. Blockers
3. Availability
4. ETA

**Comment by phuonguvan** [comment]
@ememercy21 Hey! This is the design team lead and I was wondering if you are still working on this project. Bonnie told me to update you guys so let me know as soon as possible!

**Comment by ExperimentsInHonesty** [comment]
@ememercy21 we never got an update on this issue from you, so we are assuming you are off the project.  If you do want to rejoin, just message us in the slack channel

**Comment by ExperimentsInHonesty** [comment]
I QAed this sheet, it is ready for dev leads to make good first and second issues out of it.

**Comment by ExperimentsInHonesty** [comment]
Each of the project files have alt text that will need to be revised so that later we can use it for if the logos appear by themselves with no text.  e.g., 100Automations alt should be changed from
```
alt: '100 Automations logo depicted as a 1 and two intermeshed gears surrounded by a open left bracket, forward slash and closing right bracket to indicate the 100 automations are in code, followed by the word Automations.
```
 to 
```
100 Automations
```

But in order for it not to appear on the projects page https://www.hackforla.org/projects/ we only need to revise these lines (88 & 89) from the .js file where it pulls the alt text from the project.md file https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js

>  {%- if project.alt -%},
> "alt": `{{ project.alt }}`

and replace with 
```
"alt": `""`
```

At least thats what I think should happen.  I would like a second set of eyes on this, and for it to be made into an issue.

**Comment by MattPereira** [comment]
I tested the proposed change to `current-projects.js` on my local env and it works as intended. 

Created the following issue:
- #3861

**Comment by ExperimentsInHonesty** [comment]
last item before this can be closed and moved manually to done is to make an epic issue to create all of the issues for each project file to have its alt updated to say the name of the project only

**Comment by MattPereira** [comment]
Epic issue to track update of alt values for every markdown file in the _projects collection:
- #3866
